"Error: private repository" on Releases pages

### Description

Octotree complains that repository is private on every Releases page. It makes no sense since I don't have tokens for all repos anyway.

Random example: https://github.com/sharkdp/fd/releases

### Environment (if bug)

* Octotree version: 3.0.8
* Browser & version: Chrome 75.0.3770.142
* OS & version: Linux
* Screenshot, if any (drag an image here)
![image](https://user-images.githubusercontent.com/144717/61554834-8c600580-aa66-11e9-9dc8-94f004660823.png)

* Console error log below this line
```
content.js:2117 TypeError: Cannot read property 'addEventListener' of null
    at forked_to_init (content.js:6733)
    at features_run (content.js:2112)
    at content.js:2158
    at content.js:2073
    at onAjaxedPagesRaw (content.js:2068)
features_run @ content.js:2117
async function (async)
features_run @ content.js:2112
(anonymous) @ content.js:2158
(anonymous) @ content.js:2073
onAjaxedPagesRaw @ content.js:2068
async function (async)
onAjaxedPagesRaw @ content.js:2066
onAjaxedPages @ content.js:2071
add @ content.js:2158
async function (async)
add @ content.js:2124
(anonymous) @ content.js:6747
__webpack_require__ @ content.js:12
(anonymous) @ content.js:64
(anonymous) @ content.js:65
content.js:2116 ❌ release-download-count
content.js:2117 Error: Personal token required for this feature
    at content.js:2306
    at async getAssetsForTag (content.js:6456)
    at async release_download_count_init (content.js:6484)
    at async features_run (content.js:2112)
```

--- a/src/adapters/github.js
+++ b/src/adapters/github.js
@@ -135,6 +135,8 @@ class GitHub extends PjaxAdapter {
     const branch =
       // Pick the commit ID as branch name when the code page is listing tree in a particular commit
       (type === 'commit' && typeId) ||
+      // Pick 'master' as branch name when viewing repo's releases or tags
+      ((type === 'releases' || type === 'tags') && 'master') ||
       // Pick the commit ID or branch name from the DOM
       branchFromSummary ||
       ($('.overall-summary .numbers-summary .commits a').attr('href') || '').replace(

